api_v3/services/BaseEntryService.php

```diff
@@ -1025,7 +1025,7 @@ function getPlaybackContextAction($entryId, KalturaPlaybackContextOptions $conte
 				throw new KalturaAPIException(KalturaErrors::FLAVOR_ASSET_ID_NOT_FOUND, $contextDataParams->flavorAssetId);
 		}
 
-		$contextDataHelper = new kContextDataHelper($dbEntry, $this->getPartner(), $asset);
```

Comment:
Now that you changed the order in the ctor this is not correct.
Also, please note that this class is initialized from other location as well
So if you add a mandatory param you need to add it in all places.